components/cam/src/physics/crm/sam/MICRO_SAM1MOM/micro_params.F90

```diff
@@ -35,10 +35,17 @@ module micro_params
   real(crm_rknd), parameter :: b_grau = 0.5  ! Fall speed exponent for graupel
 
   ! Autoconversion
+#ifdef CLUBB_CRM   /*microphysical tuning for CLUBB*/
```

Comment:
What is this addition here for?

Reply:
I think this is probably because Xingqiu was copying over work he did on the ECP repo which still has the CLUBB code that we took out of the E3SM repo. So this should not be added.